src/node.cc

```diff
@@ -228,16 +228,31 @@ static struct {
   }
 #endif  // HAVE_INSPECTOR
 
+  void StartTracingAgent() {
+    tracing_agent = new tracing::Agent();
```

Comment:
Yes, I've done both of these things in b93584b. In the process, I also spotted another direct call to `tracing_agent->Stop()` and replaced it with a call to `v8_platform.StopTracingAgent()`.